crm_request/models/mail_compose_message.py

```diff
@@ -38,14 +40,61 @@ def send_mail(self, auto_commit=False):
     @api.multi
     def onchange_template_id(self, template_id, composition_mode, model, res_id):
         """
-        Append the quote of previous e-mail to the body of the message.
+        - Append the quote of previous e-mail to the body of the message.
+        - Add gift payment slips for "Gift/Payment slip" and "Christmas Gift Fund"
         """
         result = super().onchange_template_id(
             template_id, composition_mode, model, res_id
         )
+
+        # Add reply qyote
```

Comment:
Little typo in the comment.
```suggestion
        # Add reply quote
```